docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-page-options/cgdp-page-options.ts

```diff
@@ -0,0 +1,58 @@
+import { trackOther } from '../../core/analytics/eddl-util';
+
+/**
+ * Tracks a language toggle click.
+ * @param linkText The language of the link.
+ * @param location Location of the page options either top or bottom of page.
+ */
+const track = (linkText: string, location: string) => {
+	trackOther('PreHeader:LinkClick', 'PreHeader:LinkClick', {
```

Comment:
This code is for language toggle, page options have different event name `PageOptions:LinkClick`- please refer to to AC